columnflow/tasks/framework/mixins.py

```diff
@@ -786,6 +786,12 @@ class PlotMixin(AnalysisTask):
         description="string parameter to define the y-axis scale of the plot in the upper panel; "
         "choices: NO_STR,linear,log; no default",
     )
+    process_lines = law.CSVParameter(
```

Comment:
Just a note:

At some point we should remove the parameters again from the basic plot mixin, but they should rather be attached to a more specific plot tasks. E.g. there's the "skip_ratio" parameter (or now "process_lines") that are good to have for 1D variable plots, but the PlotMixin should also serve as a base for completely different tasks that might not have these features. I will open an issue to document this.

But for now, I think it's ok since these rather specific parameters already exist.